app/models/turbo/streams/tag_builder.rb

```diff
@@ -229,20 +229,24 @@ def prepend_all(targets, content = nil, **rendering, &block)
   end
 
   # Send an action of the type <tt>name</tt> to <tt>target</tt>. Options described in the concrete methods.
-  def action(name, target, content = nil, allow_inferred_rendering: true, **rendering, &block)
+  def action(name, target, content = nil, method = nil, allow_inferred_rendering: true, **rendering, &block)
     template = render_template(target, content, allow_inferred_rendering: allow_inferred_rendering, **rendering, &block)
 
-    turbo_stream_action_tag name, target: target, template: template
+    turbo_stream_action_tag name, target: target, template: template, **attributes_from_method(method)
```

Comment:
I think there's no need for `attributes_from_method`, could simply be replaced by `, method: method`

Reply:
DONE